### DISCUSSION TITLE
q-date format problem

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#9946] [General - Components / Directives / etc]
I've been using date format YYYY-MM-DD in my app. Now trying to add a q-date widget and finding that it does not highlight the selected date unless I change the format to YYYY/MM/DD using slashes instead of dashes.
Is there a way to customize q-date or am I forced to use the slash format?

**Comment by stefanvanherwijnen** [comment]
Have a look at https://github.com/quasarframework/quasar/issues/3214

**Comment by fairking** [comment]
@CrawfordW please find the following example: https://github.com/quasarframework/quasar/discussions/16991 It should solve your issue.

**Reply by CrawfordW** [reply]
This issue was solved when Quasar added the 'mask' prop to the Date Picker.
I just added mask="YYYY-MM-DD".
